src/wallet/load.cpp

```diff
@@ -41,10 +41,27 @@ bool VerifyWallets(interfaces::Chain& chain, const std::vector<std::string>& wal
 
     chain.initMessage(_("Verifying wallet(s)...").translated);
 
+    // For backwards compatibility if an unnamed top level wallet exists in the
+    // wallets directory, include it in the default list of wallets to load.
+    if (!gArgs.IsArgSet("wallet")) {
```

Comment:
re: https://github.com/bitcoin/bitcoin/pull/15454#discussion_r490224007

> Would it better to do:
> 
> ```diff
> -    if (!gArgs.IsArgSet("wallet")) {
> +    if (!(gArgs.GetBoolArg("wallet", false)) {
> ```

This would be bad because it would cause a `-nowallet` setting to load the default wallet.